examples/two-columns-layout/src/functions/populate.ts

```diff
@@ -0,0 +1,37 @@
+import { AutocompleteReshapeFunction } from './AutocompleteReshapeFunction';
+import { normalizeReshapeSources } from './normalizeReshapeSources';
+
+type PopulateOptions = {
+  mainSourceId: string;
+  limit: number;
+};
+
+export const populate: AutocompleteReshapeFunction<PopulateOptions> = ({
```

Comment:
Can you explain what this function does? This is unclear to me and could be difficult without context in the future.